ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-next__414.yaml

```diff
@@ -49,6 +49,42 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:golang-1.19
   to: knative-serving-webhook
+- dockerfile_path: openshift/ci-operator/knative-perf-images/dataplane-probe/Dockerfile
```

Comment:
Probably for main we should skip the image build by changing the target so that release-next is not touched.cc @ReToCode